frontend/integration-tests/tests/monitoring.scenario.ts

```diff
@@ -36,19 +36,19 @@ describe('Monitoring: Alerts', () => {
   it('filters Alerts by name', async () => {
     await monitoringView.wait(until.elementToBeClickable(crudView.nameFilter));
     await crudView.nameFilter.sendKeys(testAlertName);
-    expect(monitoringView.firstListLinkById('alert-resource-link').getText()).toContain(
+    expect(monitoringView.firstElementByTestId('alert-resource-link').getText()).toContain(
```

Comment:
This is too general a utility to be in `monitoringView`. We'd want to move it somewhere else. Maybe protractor.conf.ts

Reply:
ok, fixed